library/src/main/scala/scala/scalajs/js/Math.scala

```diff
@@ -278,4 +278,79 @@ object Math extends Object {
    * MDN
    */
   def tan(x: Double): Double = native
+
+  /** <span class="badge badge-ecma6" style="float: right;">ECMAScript 6</span>
+    * The Math.log1p() function returns the natural logarithm (base e) of
+    * 1 + a number
+    *
+    * @return The natural logarithm (base e) of 1 plus the given number.
+    *         If the number is less than -1, NaN is returned.
+    * MDN
+    */
```

Comment:
Er ... I think you overlooked the alignment of the stars (:P), which is actually also inconsistent with the rest of this file. They're supposed to align with the first of the two stars of `/**`.